02_programming_fundamentals/08_this_day_1/05_phone_production/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "05_phone_production",
+  "version": "1.0.0",
+  "description": "",
+  "main": "05_phone_production.js",
+  "scripts": {
+    "test": "eslint . && jest"
+  },
+  "author": "",
+  "license": "ISC",
+  "jest": {
+    "noStackTrace": true
+  },
+  "dependencies": {
+    "jest": "^23.6.0",
+    "shelljs": "^0.8.2"
+  }
```

Comment:
When the `test` script starts with `eslint .`, you need to add 
```
  "eslintIgnore": [
    "__tests__"
  ]
```

to the `package.json` for `eslint` to ignore the test directory